### ISSUE TITLE
It would be useful to tell the user how to get to the interactive report screen shown on this page.

### ISSUE THREAD

**Issue by JTWWI** [MicrosoftDocs/dynamics-365-unified-operations-public#4585]
That way once we fight through this tutorial we can figure out if our changes worked. :)

**Comment by twheeloc** [comment]
@JTWWI, what topic are you referring to?

**Comment by JTWWI** [comment]
At page: https://learn.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/analytics/expand-app-suite-report-data-sets
in section https://learn.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/analytics/expand-app-suite-report-data-sets#after-custom-solution
it shows a form with a history section that seems to expand.  But I went through this module and I didn't have an expanding live report, I think it was due to the fact that I started with just this page, and didn't do some earlier tutorial to make the report more interactive.   
So my suggestion is to add a pre-requisites section or something that reminds the user they need to have done (some other page I haven't found yet) an earlier tutorial to have this page work.

Note it's confusing to find 'fleet management' tutorial stuff scattered around a few different top level documentation structures.
The page above is in a top level section called "Dynamics 365 Commerce" but other Fleet Management tutorials can be found in the top level documentation section called "Development and administration for finance and operations apps."
